data-sources/gitlab/config.yml

```diff
@@ -0,0 +1,17 @@
+id: otel-gitlab-integration
```

Comment:
We no longer need this data source since we will be using an install plan instead